docs/src/models/atmodel.rst

```diff
@@ -0,0 +1,9 @@
+:mod:`models.atmodel` -- Author-topic models
+======================================================
+
+.. automodule:: gensim.models.atmodel
+    :synopsis: Author-topic models
+    :members:
```

Comment:
The code appears to generate sphinx documentation properly, and appears in the API reference as well.